### PR TITLE
Configure Travis for automated testing

### PR DESCRIPTION
**Pull request by abhigyank**
Added .travis.yml to currently test core.py

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - "nightly" # currently points to 3.7-dev
+before_install:
+  - sudo apt-get update
+  - sudo apt-get -y install python3-pip python-dev
+  - sudo pip3 install -U setuptools
+  - sudo pip3 install -U virtualenvwrapper
+  - python3 -V
+  - pip3 -V
+# command to install dependencies
+install: " sudo pip3 install -r requirements.txt"
```
Comment (abinashmeher999) [inline .travis.yml]: space should be removed before `sudo`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# command to run tests
+script: sudo python sample/core.py
```
Comment (abinashmeher999) [inline .travis.yml:21]: `sudo` not required here.

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]
Everything else looks good! :+1: I have left a few comments. Please cleanup the commit history a little bit if can using `git rebase -i`. If it seems confusing, don't worry and just comment here. That's OK too but in that case I will have to squash and merge and your contribution will only count as one commit.

**Comment by abhigyank** [comment]
The travis build fails after removing sudo from script.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - sudo apt-get update
+  - sudo apt-get -y install python3-pip python-dev
+  - sudo pip3 install -U setuptools
+  - sudo pip3 install -U virtualenvwrapper
+  - python3 -V
+  - pip3 -V
+# command to install dependencies
+install: "sudo pip3 install -r requirements.txt"
+# command to run tests
+script: python sample/core.py
```
Comment (abinashmeher999) [inline .travis.yml]: Ahh I forgot. Could you try changing `python` to `python3`?

**Comment by abinashmeher999** [comment]
Still doesn't work :confused:. I will have a look at it later. You can delete the commits for the experimental changes.

**Comment by abhigyank** [comment]
It does work with sudo though.

**Review by abinashmeher999 — APPROVED** [review]

**Comment by abinashmeher999** [comment]
Shall I go ahead and merge this? Or do you want to improve the commit history?

**Comment by abhigyank** [comment]
Alright, go ahead with merging.

**Pull request merged**